ci_env.yml

```diff
@@ -0,0 +1,16 @@
+name: biomechanical-analysis-framework
+channels:
+  - conda-forge
+  - robotology
+dependencies:
+  - cmake
+  - clangxx>=14.0.0
```

Comment:
We topically use the compilers meta package to get the compiler, do you have any need specifically for clang?

Reply:
No, it was already there so I left it.